src/GitVersionCore/VersionConverters/GitVersionInfo/Templates/GitVersionInformation.cs

```diff
@@ -8,6 +8,30 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
+#if NET20 || NET35
+#define SHOULD_DEFINE_EXCLUDE_FROM_CODE_COVERAGE
+#elif NETCOREAPP1_0 || NETCOREAPP1_1
+#define SHOULD_DEFINE_EXCLUDE_FROM_CODE_COVERAGE
+#elif NETSTANDARD1_0 || NETSTANDARD1_1 || NETSTANDARD1_2 || NETSTANDARD1_3 || NETSTANDARD1_4 || NETSTANDARD1_5 || NETSTANDARD1_6
+#define SHOULD_DEFINE_EXCLUDE_FROM_CODE_COVERAGE
+#endif
```

Comment:
Why all these `#elif`'s? Couldn't one big `#if` cover all targets?

Reply:
That's just an attempt to split the very long #if line... I did something similar in the VB file, but couldn't do it in F#.

Reply:
I see.